Makefile

```diff
@@ -1154,6 +1165,16 @@ glance_kuttl: namespace input openstack_crds openstack_storage_crds deploy_clean
 	make deploy_cleanup
 	make cleanup
 
+.PHONY: barbican_kuttl_run
+barbican_kuttl_run: ## runs kuttl tests for the barbican operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${BARBICAN_KUTTL_CONF} ${BARBICAN_KUTTL_DIR}
+
+.PHONY: barbican_kuttl
+barbican_kuttl: namespace input openstack_crds openstack_storage_crds deploy_cleanup mariadb mariadb_deploy keystone keystone_deploy barbican_deploy_prep barbican ## runs kuttl tests for the barbican operator. Installs openstack and openstack-storage crds, mariadb, keystone and barbican operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
```

Comment:
Judging from the controller files from barbican-operator, the only dependency that will be need is `infra`, but will need later confirmation as it seems the operator it's in a very early stage (there are no images pushed to quay yet).